styles.css

```diff
@@ -14,26 +14,61 @@ header, footer{
   flex: 0 0 100%;
   background:#eee;
 }
+
+header>h1{
+  margin: 0px;
+  color: red;
+}
+
+footer>h3{
+  margin: 0px;
+  color: red;
+}
+
+main>h2{
+  margin: 0px;
+  color: red;
+}
+
+aside>h2{
+  margin: 0px;
+  color: red;
+}
+
```

Comment:
👍 This works, but also an opportunity to make use of some pseudo-selectors to help DRY up this code